src/main/resources/swagger.api/schema/resource/request/InstanceRequest.json

```diff
@@ -206,14 +206,6 @@
             "type": "object",
             "$ref": "../common/IdField.json"
           }
-        },
-        "inventoryId": {
```

Comment:
Removed inventoryId and srsId from the request DTO as UI will not send these values.
May be we can remove these from the response DTO also. What do you think?

Reply:
I think we should ask frontend team if they somehow use inventoryId and srsId on their side

Reply:
Front end wont use it. I have the front end code.

Reply:
I think we better keep it in the Response, to be able to debug/investigate something

Reply:
Ok. I will replace the "srsId" and "inventoryId" from the response and replace it with a new "instanceMetadata" DTO.

Reply:
Done